.idea/dbnavigator.xml

```diff
@@ -0,0 +1,458 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
are .idea/dbnavigator.xml and .idea/misc.xml are required ?

Reply:
no sorry, they got committed by mistake, will delete them. Thanks @shreedhar-hmcts